src/data/pokemon/species_info/gen_1_families.h

```diff
@@ -278,6 +278,24 @@ const struct SpeciesInfo gSpeciesInfoGen1[] =
         .iconPalIndex = 4,
         SHADOW(2, 6, SHADOW_SIZE_XL_BATTLE_ONLY)
         FOOTPRINT(Venusaur)
+    #if OW_POKEMON_OBJECT_EVENTS && OW_BATTLE_ONLY_FORMS
```

Comment:
`OVERWORLD`'s definition already handles `OW_POKEMON_OBJECT_EVENTS`, so you only need `OW_BATTLE_ONLY_FORMS` here.

Reply:
hmm, if that's the case, should i just put `OW_BATTLE_ONLY_FORMS` in the definition too?

Reply:
wait, no, answered my own question lol.